### DISCUSSION TITLE
Pupil tracking

### DISCUSSION THREAD

**Discussion by ederancz** [sn-lab/MouseGoggles#12] [Ideas]
Hello,

Did you ever consider some pupil-tracking solutions? 
I would appreciate your thoughts. We are thinking of integrating a hot mirror into the oculars so can use IR imaging, but you probably thought more deeply about this.  

Thanks,
Ede

**Comment by misaacson01** [comment]
Hi, yes we have considered it! We also began designing an eyepiece with a slot for an IR hot mirror, though it's been tabled for now since we have other higher priority projects. It might be challenging to implement since there is so little space to work with, and since the pupil is very close to the Fresnel lens. But one idea we had as for positioning the hot mirror is just to have it lay parallel with the display, and position a (pinhole) camera on the same side as the mouse's eye, as close to the eye as possible. You could do IR illumination as well from a different side of the mouse's eye (also pointed in the same direction as the eye). 

Keep us posted on your progress with this!
